pages.de/common/vimtutor.md

```diff
@@ -0,0 +1,12 @@
+# vimtutor
+
+> Vim-Tutor zum Erlernen grundlegender Vim-Befehle.
+> Weitere Informationen: <https://manned.org/vimtutor>.
+
+- Öffne den Vim-Tutor in englischer|französischer|deutscher|... Sprache:
+
+`vimtutor {{en|fr|de|...}}`
```

Comment:
```suggestion
- Öffne den Vim-Tutor in der gegebenen Sprache (de, en, fr, ...):

`vimtutor {{sprache}}`
```

Reply:
This sounds reasonable as vimtutor only accepts the language code and not the language itself (at least on my machine).